### PR TITLE
Update : JSON response on GetPods + Job Creation

### PR DESCRIPTION
**Pull request by saisona**
- Now the /pods is responding with all Pods items in an AWS cluster.
 - creation of /migrate that is creating a fake inarix job

 @cesumilo , would you prefer that we say that Inarix will use my private repo as application ? Or do we make this
 repo public and we create a private repo with Inarix that uses my OpenSource go-feather-slack go package ?

 FYI : warning, we still need to add the GoogleAuthentication so we will
 be able to create some pods on both AWS and GCP clusters

Closing #1

### CODE REVIEW

**Comment by saisona** [comment]
> > * Now the /pods is responding with all Pods items in an AWS cluster.
> > * creation of /migrate that is creating a fake inarix job
> > 
> > @cesumilo , would you prefer that we say that Inarix will use my private repo as application ? Or do we make this
> > repo public and we create a private repo with Inarix that uses my OpenSource go-feather-slack go package ?
> > FYI : warning, we still need to add the GoogleAuthentication so we will
> > be able to create some pods on both AWS and GCP clusters
> > Closing #1
> 
> I cannot take this decision alone. If any of the software has been made during office hours, it's owned by Inarix. However, I let you discuss with Artemis about it.

None of all those lines has been made during office hours for that reason :), since then I completely agree with you about the fact that it could be good to ask to Artemis.
However I thought that you could give me a base line that we could follow, so whether I chose to keep thinking about making it open source or I'm starting to build it as it will be used only for Inarix purpose and then I'll have to talk about it with Artemis about what does he wants to do.

**Comment by cesumilo** [comment]
> > > * Now the /pods is responding with all Pods items in an AWS cluster.
> > > * creation of /migrate that is creating a fake inarix job
> > > 
> > > @cesumilo , would you prefer that we say that Inarix will use my private repo as application ? Or do we make this
> > > repo public and we create a private repo with Inarix that uses my OpenSource go-feather-slack go package ?
> > > FYI : warning, we still need to add the GoogleAuthentication so we will
> > > be able to create some pods on both AWS and GCP clusters
> > > Closing #1
> > 
> > 
> > I cannot take this decision alone. If any of the software has been made during office hours, it's owned by Inarix. However, I let you discuss with Artemis about it.
> 
> None of all those lines has been made during office hours for that reason :), since then I completely agree with you about the fact that it could be good to ask to Artemis.
> However I thought that you could give me a base line that we could follow, so whether I chose to keep thinking about making it open source or I'm starting to build it as it will be used only for Inarix purpose and then I'll have to talk about it with Artemis about what does he wants to do.

Make it open source, it's a good exercice for your futur project. I don't know what will say Inarix, however, I already know some points where you're screw. Let's talk about it in private hehe.

File: src/go-feather-slack-app/server/main.go [hunk 1]
```diff
@@ -19,37 +21,83 @@ import (
...
-		pods, err := self.manager.GetPods(r.Form.Get("namespace"))
+
+		log.Printf("Selected namespace for pod introspection is %s", FormValues.Namespace)
+		pods, err := self.manager.GetPods(FormValues.Namespace)
 		if err != nil {
 			w.WriteHeader(500)
 			fmt.Fprintln(w, err.Error())
 		}
-		json.Marshal(pods.Items)
+		marchalled_kubernetes_items, err := json.Marshal(pods.Items)
```
Comment (cesumilo) [inline src/go-feather-slack-app/server/main.go:69]: You use snake case here. You're using camel case usually so I guess it's a mistake.

**Review by cesumilo — CHANGES_REQUESTED** [review]
Some consistency issues and also what would be nice is to extract constant string into constants 👍

File: src/go-feather-slack-app/server/main.go (hunk 1 above)
Comment (saisona) [reply src/go-feather-slack-app/server/main.go:69]: nice good idea :)

**Comment by saisona** [comment]
> Some consistency issues and also what would be nice is to extract constant string into constants 👍

@cesumilo what do you mean by extract constant strings

**Pull request merged**